app/repository_models/collection.rb

```diff
@@ -55,4 +56,10 @@ def generate_derivatives
   def generate_thumbnail_url
     "/downloads/#{self.representative}?datastream_id=thumbnail"
   end
+
+  def owner_name(edit_user)
+    if user = User.find_by_email(edit_user)
+     user.inverted_name.titleize
```

Comment:
titleize capitalizes the first letter of any word in a string. I figured it should be capitalized properly.